content/en/blog/_posts/2024-07-xx-kube-scheduler-simulator/index.md

```diff
@@ -0,0 +1,135 @@
+---
+layout: blog
+title: "Introducing Kube-scheduler-simulator"
+date: 2024-07-xx
+slug: introducing-kube-scheduler-simulator
+author: Kensei Nakada (Tetrate)
+---
+
+The Kubernetes Scheduler is a crucial control plane component that determines which Node a Pod will run on. 
```

Comment:
As a convention, we use 'Node' when we refer to the Node resource defined by the API, and we use 'node' as a regular term when we refer to the machine where workloads are
scheduled and run.

This is a style recommendation for all docs on the site, though not a strict one for
logs. Please check the doc when proposing your next commit to this PR.